src/main/resources/assets/goodgenerator/lang/en_US.lang

```diff
@@ -16,6 +16,8 @@ essentiaCell.0.name=Novice Essentia Diffusion Cell
 essentiaCell.1.name=Adept Essentia Diffusion Cell
 essentiaCell.2.name=Master Essentia Diffusion Cell
 essentiaCell.3.name=Grandmaster Essentia Diffusion Cell
+essentiaCell.4.name=Biological Activity Essentia Diffusion Cell
```

Comment:
Biological activity shouldn't have anything to do with thaumcraft, perhaps something along the lines of "Void Encrusted Essentia Diffusion Cell" or something?

Reply:
It's not thought out yet and is open to discussion. It's just used to keep the game from being essentiaCell.name like this